### ISSUE TITLE
[Accessibility issue] "Everything looks good, you can submit now !" message should be read out once the edit boxes have been filled with necessary format details.

### ISSUE THREAD

**Issue by harshamulpuri2913** [ampproject/amphtml#3958]
## Short description of your issue: within iOS device (IPAD / IPOD), observe "Everything looks good, you can submit now !" message  is not being read out by voice over once Name & email text boxes have been filled with necessary format

Test URL: https://amp-mk-1.herokuapp.com/examples.build/forms.amp.max.html

Pre-requisite: Open IPAD--> Settings --> General --> Accessibility --> Voiceover , Set to ON
## How do we reproduce the issue?
1. Logon to IPAD and open the safari browser
2. Open test URL provided above
3. Under "Subscribe to our weekly newsletter (non-xhr)", please enter name as "Y" & email value as "Y@gmail.com"
4. Observe as we start entering the values, some message's that required fields are not filled are displayed and once all details (eg: format of email is satisfied) a message "Everything looks good, you can submit now !" is displayed

Actual results: 
1. Observe if both Name and email values are provided as input, message "Everything looks good, you can submit now !" is displayed

Expected result: 
1. Once the necessary formats are provided as input, message should be read out by voiceover as "Everything looks good, you can now subscribe !" should be displayed and upon click on subscribe, user should be able to subscribe.
## What browsers are affected? Safari , Chrome

All browsers? Some specific browser? What device type?
Device details: IPAD 
iOS version: 9.3.2 (13F69)

Attachments: 
![img_0045](https://cloud.githubusercontent.com/assets/20247457/16671290/7a0838f2-4453-11e6-8a76-d193cdc32568.PNG)

**Comment by mkhatib** [comment]
This message is displayed by the publisher using CSS. AMP doesn't know of that element and it's just a div that's shown with CSS.

We can address providing a better mechanism when we get to implement custom validation messages and UI.

**Comment by adelinamart** [comment]
@mkhatib is this still an issue? Thanks

**Comment by mkhatib** [comment]
Custom validation should be a better accessible approach to show error messages. This message is only shown through CSS classes. Closing this, we launched custom-validation couple week ago.
